src/Notifications/NotificationCenterController.cs

```diff
@@ -349,10 +356,6 @@ protected virtual void Dispose(bool disposing)
         /// </summary>
         public void Dispose()
         {
-            if (initState == AsyncMethodState.Started)
```

Comment:
Do we still need the `AsyncMethodState.Started` option?

Reply:
I have some more changes to completely remove the AsyncMethodState enum

Reply:
We do not need that anymore